### ISSUE TITLE
[Issue]: Missing episodes are not deleted and can't be deleted

### ISSUE THREAD

**Issue by VigibotDev** [jellyfin/jellyfin#7207]
### Please describe your bug

When I add a new episodes, the "Missing episode" metadata is not deleted and I can't delete.
I have a lot of TV show with this problem and delete entire database make some days of scanning.
How to refresh only missing episode without delete entire database for shows ?

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by michaelkrieger** [comment]
Yes. Either we need to be able to purge all missing episodes (and they can be recreated) or rescan and it should remove them automatically if the episode exists.

**Comment by VigibotDev** [comment]
I removed TVDB for this... Rescan never worked, only ".ignore" (or moving out of jellyfin library) the directory and rescan, add and rescan...

**Comment by rexzhang** [comment]
I have the same problem

**Comment by Rhyss06** [comment]
This is still a problem. Anyone found a work around?

**Comment by VigibotDev** [comment]
Sorry I removed entirely this feature. Too mutch entropy I can't permit this on my server...

**Comment by FabrizioMFE** [comment]
Reviving this as it's still an issue https://i.imgur.com/xinBfzo.jpeg Missing Episode Fetcher metadata is disabled and library has been rescanned and redownloaded metadata (overwrite) multiple times.

**Comment by adamflorizone** [comment]
I have the exact same issue as @FabrizioMFE. There seems to be an issue with duplicate "missing" when the the episodes are in fact also there.

**Comment by Ithiloneth** [comment]
I've done got me 40.000 missing episodes.
Can't get rid of them though I've several days past disabled the Missing Episodes Fetcher and scan daily.

Deleting the library and starting fresh is not a tempting proposition - I put work into selecting cover art and fixing bad IDs.

They all seem to be nicely tagged and index-able, all that's missing is a command to delete the entries once they are selected for.

Anyone able to fix such a solution?

**Comment by PCJones** [comment]
The problem seems to be around here
https://github.com/jellyfin/jellyfin/blob/60881eaa5981ffced3615fbff079871ed24c70e4/MediaBrowser.Providers/TV/SeriesMetadataService.cs#L104

**Comment by duracell** [comment]
Same problem still exists.

**Comment by CZEMacLeod** [comment]
Would it be possible to add a rescan option specifically for this - check each missing episode and, if it exists, remove it, rather than rescanning everything or rebuilding the entire database?

**Comment by CHRONiCiTY** [comment]
I'm having this issue as well. Really annoying. Shows that have their folders completely deleted from the harddrive still show up in Jellyfin no matter how much I rescan the library.

**Comment by Dan6erbond** [comment]
I also have this issue and it's causing inconvenience when browsing a show. Are there any plans to fix this?

**Comment by PCJones** [comment]
Still not fixed.
![grafik](https://github.com/jellyfin/jellyfin/assets/377223/1a7d6280-b0ef-49e9-a3be-c28379358841)

**Comment by Shadowghost** [comment]
This might require either an replace all metadata or find missing metadata kind of scan to clean up properly

**Comment by PCJones** [comment]
> This might require either an replace all metadata or find missing metadata kind of scan to clean up properly

I agree, allowing us to delete all metadata would at least be a quick fix. But afaik there is no way to do that.

**Comment by PCJones** [comment]
Possibly fixed? https://github.com/jellyfin/jellyfin/pull/13218

**Comment by Shadowghost** [comment]
Fixed by #13218
